### ISSUE TITLE
Provide an option to bypass the Omniorb name service.

### ISSUE THREAD

**Issue by jmirabel** [Gepetto/gepetto-viewer-corba#103]
Here is a simple Python example of how to do this.
- server.idl
```
module server {
  interface foo {
    string hello (  );
  };
};
```
- run_server.py:
```py
import CORBA, server, server__POA

class Server(server__POA.foo):
    def hello(self):
        return "hello"

orb = CORBA.ORB_init(['-ORBendPoint', 'giop:tcp:localhost:12321'], CORBA.ORB_ID)
poa = orb.resolve_initial_references("omniINSPOA")

servant = Server (  )
poa.activate_object_with_id("exampleServer", servant)
print orb.object_to_string(poa.servant_to_reference (servant))

poa._get_the_POAManager().activate(  )
orb.run(  )
```
- run_client.py
```py
import CORBA, server, sys
orb = CORBA.ORB_init(sys.argv , CORBA.ORB_ID)
o = orb.string_to_object("corbaloc::localhost:12321/exampleServer")
o = o._narrow(server.foo)
print o.hello(  )
```

Compile server.idl with `omniidl -bpython server.idl`, then run `python run_server.py` and `python run_client.py`

**Comment by jmirabel** [comment]
Related to https://github.com/stack-of-tasks/pinocchio/issues/657

**Comment by jmirabel** [comment]
Fixed in #104
